lib/cartopy/mpl/geoaxes.py

```diff
@@ -417,6 +417,34 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
+    def tissot(self, rad_km=5e5, n_samples=80, lon_n=10, lat_n=6, **kwargs):
+        """
+        Adds Tissot's indicatrices to the current axes with the geodesic class.
+
+        Kwargs:
+
+            * rad_km - The radius in km of the the circles to be drawn.
+
+            * n_samples - Integer number of points sampled around the
+              circumference of each circle.
+
+            * lon_n - Number of circles in the longitudinal direction.
+
+            * lat_n - Number of circles in the latitudinal direction.
+
+        """
+        geod = cartopy.geodesic.Geodesic()
+        geoms = []
+
+        for lat in np.linspace(-80, 80, lat_n):
```

Comment:
I was actually considering suggesting that rather than specifying ```lon_n``` ```lat_n``` we simply put the circles ```rad_km``` apart, working out from the equator and central meridian. 

I haven't looked at how well it would work, but I'd also consider putting the tissot on each of the projections in the [projection list](http://scitools.org.uk/cartopy/docs/latest/crs/projections.html). (see https://github.com/SciTools/cartopy/blob/master/docs/make_projection.py#L97)